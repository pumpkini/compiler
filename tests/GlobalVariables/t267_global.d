int x;
int[] y;
int[] t;
void anotherFunction(int n){
	Print("global value of x is ", x);
}
void assignValues(int n){
	bool x;
	x = true;
	Print("local value of x is ", x);
}
void main(){
    y = NewArray(100, int);
    t = NewArray(100, int);
	x = 100;
	assignValues(99);
	anotherFunction(99);
}