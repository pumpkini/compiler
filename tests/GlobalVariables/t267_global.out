local value of x is true
global value of x is 100
